src/Components/GoalForm/GoalForm.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createUserGoal } from '../../actions';
+import './GoalForm.css';
+
+class GoalForm extends Component {
+  constructor(props) {
+    super(props);
```

Comment:
good to have the goal form here. This is what we have decided to put on the sidebar, I believe